### DISCUSSION TITLE
Add "deploy to" buttons to README

### DISCUSSION THREAD

**Discussion by MekDrop** [ImpressCMS/impresscms#1041] [Ideas]
Some hosting services has a possibility to deploy project there only by clicking button and filling some simple forms. I think it would be great if in README would be a list with such services.

At current moment I found these services that has such or similar feature:
https://www.netlify.com/docs/deploy-button/
https://devcenter.heroku.com/articles/heroku-button
https://wedeploy.com/blog/one-click-deployments-with-the-wedeploy-button/
https://www.microsoft.com/developerblog/2017/01/17/the-deploy-to-azure-button/
https://console.bluemix.net/docs/services/ContinuousDelivery/deploy_button.html#deploy-button
https://installer.71m.us
https://zeit.co/docs/v2/deployments/builders/overview#when-to-use-builders

I think all of them needs special configuration files to be included in our repo. So before adding these buttons we need to create such files. But I don't think that would be hard.

**Comment by fiammybe** [comment]
For these configuration files, I would propose we have one project PER service, if that is feasible. That would allow versioning more easily

**Comment by MekDrop** [comment]
I haven't looked much there yet but it seems that some services could be described in any repo but some like Heroku needs to be included in repo from where it needs to fetch the source. So, I'm not sure yet what good bets way could be for the implementation.
